dbmanager/src/dbmanager.c

```diff
@@ -21,18 +22,131 @@ struct DB {
   dbNode* head;
 };
 
+
+/*  function to create a new entry for the database.
+ *  @key - a unique integer value.
+ *  @val - pointer to the data stored.
+ *  @return - pointer if success, NULL if failure.
+ */
+static dbNode* create_node(int key, void* val) {
+  dbNode* newNode = (dbNode*)malloc(sizeof(dbNode));
+  return newNode;
+}
```

Comment:
Should populate the node with key/val, also you should clear the pointers or use calloc